natlas-server/hooks/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set +e
+set +x
+
+NATLAS_VERSION=${DOCKER_TAG:?Specify DOCKER_TAG}
+
+if [ "$DOCKER_TAG" == "latest" ]
+then
+  NATLAS_VERSION=$SOURCE_COMMIT
```

Comment:
It'd be cool if we could use the `git describe` syntax here, since this build hook is just a bash script. But I think it won't work because docker hub does a shallow clone. The `git describe` output is a little friendlier to read and still has enough of the hash identifer to do a checkout. Maybe a later project.